packages/core/test/FramebufferSystem.tests.ts

```diff
@@ -3,93 +3,95 @@ import { MSAA_QUALITY } from '@pixi/constants';
 import { Rectangle } from '@pixi/math';
 import { expect } from 'chai';
 
-describe('FramebufferSystem', function ()
+describe('FramebufferSystem', () =>
 {
-    before(function ()
+    let renderer: Renderer;
```

Comment:
One of the benefits of using Mocha is that the scope is maintained for tests. So you can set properties on `before` or `beforeEach`. Then the tests can use that same scope. Using fat-arrow functions removes this scoping benefit. So you end up having to do a lot of variable scoping like this. It was intentional to leave the use of `function` in tests and not use fat-arrows. Could you explain why you feel this approach is beneficial?

Reply:
The main reason I used fat arrows is because jest doesn't support scoping like this

Also these added variables would always have the `any` type, so I had to keep casting the variable which was less clean